Revit_Core_Engine/Convert/Structure/FromRevit/Profile.cs

```diff
@@ -862,7 +862,11 @@ private static BH.oM.Geometry.Line MullionLine(this Mullion mullion)
         /****              Private helpers              ****/
         /***************************************************/
 
-        private static readonly BuiltInParameterGroup[] dimensionGroups = { BuiltInParameterGroup.PG_STRUCTURAL_SECTION_GEOMETRY, BuiltInParameterGroup.PG_GEOMETRY };
+#if REVIT2020 || REVIT2021 || REVIT2022
```

Comment:
This is a bit risky because you seem to be changing the behaviour for Revit 2023 and 2024, am I correct? It may look a bit like an overkill, but the code below would allow us not to even think how does this part of the code work and whether the change would affect the functionality by any means
```suggestion
#if REVIT2020 || REVIT2021 || REVIT2022 || REVIT2023 || REVIT2024
```